docker/auth/realm.json

```diff
@@ -2259,4 +2274,5 @@
     "clientPolicies": {
       "policies": []
     }
-  }
\ No newline at end of file
+  }
+  
```

Comment:
Could you fix EOF

Reply:
It was badly indentated, I have a parser on it now.
[9a572d1](https://github.com/numerique-gouv/people/pull/13/commits/9a572d1c56b61cd474e25d163a27aa63f99a2f17)